src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx

```diff
@@ -32,54 +32,33 @@ import kubernetesCke from 'images/kubernetes_logo_cke.png'
 
 import kubernetesEks from 'images/kubernetes_logo_eks.png'
 
-The easiest way to install the Kubernetes integration is to use our guided install to generate a manifest. It bundles not just the integration DaemonSets, but also other New Relic Kubernetes configurations, like [Kubernetes events](/docs/integrations/kubernetes-integration/kubernetes-events/install-kubernetes-events-integration), [Prometheus OpenMetrics](/docs/integrations/prometheus-integrations/get-started/new-relic-prometheus-openmetrics-integration-kubernetes), and [New Relic log monitoring](/docs/logs).
+The easiest way to install the Kubernetes integration is to use our guided install to generate a manifest. It bundles not just the integration DaemonSets, but also other New Relic Kubernetes configurations, like [Kubernetes events](/docs/integrations/kubernetes-integration/kubernetes-events/install-kubernetes-events-integration), [Prometheus Agent](/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent/), and [New Relic log monitoring](/docs/logs).
```

Comment:
```suggestion
The New Relic Kubernetes integration gives you full observability into the health and performance of your environment by leveraging the New Relic infrastructure agent to collect metrics from your cluster alongside various other New Relic integrations such as the [Kubernetes events integration](/docs/integrations/kubernetes-integration/kubernetes-events/install-kubernetes-events-integration), the [Prometheus Agent](/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent/), and the [New Relic log plugin](/docs/logs).
```

Reply:
This sentence is toooo long. I've added a period:
"...by leveraging the New Relic infrastructure agent. This agent collects metrics..."